### PR TITLE
Added the governance template

### LINKED ISSUE #248: Working Group governance model & charter
From the Collaborator Summit in Austin, TX, it emerged the fact that the Streams WG does not have a proper charter, even if it has been chartered when the Foundation has been created. I think we should fill this in, and officialise ourselves. The alternative is to move to a less formal group, a "team".

As I understood, a chartered wg has a a delegation from tsc/ctc.

@williamkapke can probably clarify some of this, and help us with the proper paperwork.

As an example, see https://github.com/nodejs/security-wg/pull/9.

### PR DESCRIPTION
**Pull request by mcollina**
Fixes #248.

Folks, let's suggest edits and ratify this.

cc @nodejs/streams @williamkapke

### CODE REVIEW

File: GOVERNANCE.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+No more than 1/3 of the WG members may be affiliated with the same
+employer.  If removal or resignation of a WG member, or a change of
+employment by a WG member, creates a situation where more than 1/3 of
+the WG membership shares an employer, then the situation must be
+immediately remedied by the resignation or removal of one or more WG
+members affiliated with the over-represented employer(s).
+
+### WG Meetings
+
+The WG meets weekly on a Google Hangout On Air. A designated moderator
```
Comment (williamkapke) [inline GOVERNANCE.md]: Probably need to change this to fit this WGs meetings...right?

File: GOVERNANCE.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+When an agenda item has appeared to reach a consensus the moderator
+will ask "Does anyone object?" as a final call for dissent from the
+consensus.
+
+If an agenda item cannot reach a consensus a WG member can call for
+either a closing vote or a vote to table the issue to the next
+meeting. The call for a vote must be seconded by a majority of the WG
+or else the discussion will continue. Simple majority wins.
+
+Note that changes to WG membership require unanimous consensus.  See
```
Comment (williamkapke) [inline GOVERNANCE.md]: This line in this template is a bit heavy handed. Do you really require unanimous consensus? I suggest a majority vote.

File: README.md [hunk 3]
```diff
@@ -23,9 +23,26 @@ Full documentation may be found on the [Node.js website](https://nodejs.org/dist
...
+* Addressing stream issues on the Node.js issue tracker.
+* Authoring and editing stream documentation within the Node.js project.
+* Reviewing changes to stream subclasses within the Node.js project.
+* Redirecting changes to streams from the Node.js project to this
+  project.
+* Assisting in the implementation of stream providers within Node.js.
+* Recommending versions of `readable-stream` to be included in Node.js.
+* Messaging about the future of streams to give the community advance
+  notice of
+  changes.
```
Comment (williamkapke) [inline README.md]: extra `\n` here

File: README.md [hunk 4]
```diff
@@ -23,9 +23,26 @@ Full documentation may be found on the [Node.js website](https://nodejs.org/dist
...
 Node you, or the users of your libraries are using, use **readable-stream** *only* and avoid the *"stream"* module in Node-core, for background see [this blogpost](http://r.va.gg/2014/06/why-i-dont-use-nodes-core-stream-module.html).
 
-As of version 2.0.0 **readable-stream** uses semantic versioning.  
-
-# Streams WG Team Members
+As of version 2.0.0 **readable-stream** uses semantic versioning.
+
+# Streams Working Group
+
+`readable-stream` is maintained by the Streams Working Group, who
```
Comment (williamkapke) [inline README.md]: should read: "which oversees"

**Comment by mcollina** [comment]
@williamkapke updated.

**Comment by mafintosh** [comment]
👍 and 👍 to @williamkapke's changes

**Comment by mcollina** [comment]
I will leave this open till the 27th of December and then merge. Is it ok @williamkapke?

**Comment by mcollina** [comment]
It's the 29th and no one objected, merging.

**Pull request merged**